Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY: release
+release: depend fmt lint vet test
+
+.PHONY: depend
+depend:
+	go get -t -v ./...
```

Comment:
Do we need to do this here?  What is it for?  It's hostile to the caller's GOPATH.

Reply:
There are several external dependencies in the package that need to be installed before testing. This is what the `.travis.yml` is currently doing. What would be the preferred approach, to use a local `.go` folder within the package itself? I'll need to make the same update in `verify-golint.sh` which attempts to `go get` golint if it's not already on the user's path.

Reply:
I ran a quick test, and seems like it was only to work around dependency to glog
```
$ go get -t -v ./...
github.com/golang/glog (download)
k8s.io/utils/clock
k8s.io/utils/io
github.com/golang/glog
k8s.io/utils/exec
k8s.io/utils/pointer
k8s.io/utils/clock/testing
k8s.io/utils/temp
k8s.io/utils/exec/testing
k8s.io/utils/temp/temptest
k8s.io/utils/nsenter
```